modules/parrableIdSystem.js

```diff
@@ -70,6 +70,15 @@ function isValidConfig(configParams) {
   return true;
 }
 
+function encodeBase64Safe(base64) {
```

Comment:
this should be encodeBase64UrlSafe since "url safe" is how this encoding is referenced in public documents.